modules/installation-configuration-parameters.adoc

```diff
@@ -192,7 +192,12 @@ Only IPv4 addresses are supported.
 endif::agent,bare,ibm-power,ibm-z,vsphere[]
 
 ifdef::agent,bare,ibm-power,ibm-z,vsphere[]
+Consider the following information before you configure network parameters for your cluster:
+
 * If you use the {openshift-networking} OVN-Kubernetes network plugin, both IPv4 and IPv6 address families are supported.
+** If you want a host with a non-IPv6 network configuration to access a cluster with a configured IPv6 network, you can configure the cluster's firewall to block traffic for the IPv6 protocol.
+* If you deployed nodes in an {product-title} cluster with a network that supports both IPv4 and non-link-local IPv6 addresses, with the exception of `fe80::`, configure your cluster to use a dual-stack network.
```

Comment:
The RFC documentation is confusing in that fe80: is jumbled in with link-local addresses:

![Screenshot from 2024-07-25 12-07-59](https://github.com/user-attachments/assets/478bc79b-b5ea-4a76-8965-245ef70f56fa)

https://datatracker.ietf.org/doc/rfc7404/

Stating `fe80::` as a non-LLA is correct?

Reply:
fe80:: is link-local: https://en.wikipedia.org/wiki/Link-local_address#IPv6

I think for the purposes of this doc, just saying non-link-local is sufficient. That should make it clear that the link-local address you get in ipv4 deployments is fine, but that if you're doing something more than that you need to use dual stack.

Reply:
:heavy_check_mark: